src/pages/login.svelte

```diff
@@ -12,6 +16,34 @@
     import MediaQuery from '~/components/utils/media-query.svelte'
     import Features from '~/components/elements/features.svelte'
     import UnicoveAnimated from '~/components/elements/unicove-animated.svelte'
+
+    const whalesplainerUrl = import.meta.env.SNOWPACK_PUBLIC_WHALESPLAINER_URL
+
+    async function createAccount() {
+        const accountCreator = new AccountCreator({
+            scope: 'unicove',
+            whalesplainerUrl,
+        })
+
+        const accountCreationResponse: AccountCreationResponse =
+            await accountCreator.createAccount()
+
+        if (accountCreationResponse.error) {
+            return addToast({
+                title: 'Unable to create account',
+                message: `An error occured during account creation: ${accountCreationResponse.error}!`,
+                timeout: 10000,
+            })
+        }
+
+        addToast({
+            title: 'Account created!',
+            message: `Successfully created the "${
+                (accountCreationResponse as any).sa
```

Comment:
@jnordberg Aaron and I were also wondering if you knew of a more elegant way to avoid a `Property 'sa' does not exist on type 'AccountCreationResponse'` typescript error here.

Reply:
The type is `CallbackPayload | AccountCreationErrorResponse`, the payload has sa. Maybe it's way you're checking for the error that's confusing the typescript compiler, try doing it without the return and use else

Reply:
I already tried that, it didn't make a difference.

Reply:
Hmm, not sure what's going on then. Usually typescript can infer those types easily, maybe it's because CallbackPayload comes from a dependency and it fails because of that. Casting it to any is an ok workaround here IMO